### ISSUE TITLE
baton 2.0.1 fails to build when bioconda and conda-forge in channels

### ISSUE THREAD

**Issue by kjsanger** [wtsi-npg/npg_conda#194]
```
ERROR:root:In file included from /opt/conda/conda-bld/baton-pkg_1600274766318/_build_env/x86_64-conda-linux-gnu/sysroot/usr/include/string.h:637,
ERROR:root:                 from query.c:30:
ERROR:root:query.c: In function 'prepare_obj_repl_list':
ERROR:root:query.c:287:5: error: '__builtin_strncpy' specified bound depends on the length of the source argument [-Werror=stringop-overflow=]
ERROR:root:  287 |     strncpy(path1, path, len);
ERROR:root:      |     ^~~~~~~
ERROR:root:query.c:280:18: note: length computed here
ERROR:root:  280 |     size_t len = strlen(path) + 1;
ERROR:root:      |                  ^~~~~~~~~~~~
ERROR:root:In file included from /opt/conda/conda-bld/baton-pkg_1600274766318/_build_env/x86_64-conda-linux-gnu/sysroot/usr/include/string.h:637,
ERROR:root:                 from query.c:30:
ERROR:root:query.c:288:5: error: '__builtin_strncpy' specified bound depends on the length of the source argument [-Werror=stringop-overflow=]
ERROR:root:  288 |     strncpy(path2, path, len);
ERROR:root:      |     ^~~~~~~
ERROR:root:query.c:280:18: note: length computed here
ERROR:root:  280 |     size_t len = strlen(path) + 1;
ERROR:root:      |                  ^~~~~~~~~~~~
ERROR:root:In file included from /opt/conda/conda-bld/baton-pkg_1600274766318/_build_env/x86_64-conda-linux-gnu/sysroot/usr/include/string.h:637,
ERROR:root:                 from query.c:30:
ERROR:root:query.c: In function 'prepare_obj_list':
ERROR:root:query.c:189:5: error: '__builtin_strncpy' specified bound depends on the length of the source argument [-Werror=stringop-overflow=]
ERROR:root:  189 |     strncpy(path1, path, len);
ERROR:root:      |     ^~~~~~~
ERROR:root:query.c:182:18: note: length computed here
ERROR:root:  182 |     size_t len = strlen(path) + 1;
ERROR:root:      |                  ^~~~~~~~~~~~
ERROR:root:In file included from /opt/conda/conda-bld/baton-pkg_1600274766318/_build_env/x86_64-conda-linux-gnu/sysroot/usr/include/string.h:637,
ERROR:root:                 from query.c:30:
ERROR:root:query.c:190:5: error: '__builtin_strncpy' specified bound depends on the length of the source argument [-Werror=stringop-overflow=]
ERROR:root:  190 |     strncpy(path2, path, len);
ERROR:root:      |     ^~~~~~~
ERROR:root:query.c:182:18: note: length computed here
ERROR:root:  182 |     size_t len = strlen(path) + 1;
ERROR:root:
```

Caused by changes to build tools, I think. Also ... why/how is is the build running as root?! The container steps down from root to the conda user on startup ...

**Comment by kjsanger** [comment]
Solution is to move this back out of red-recipes - it doesn't depend in iRODS 4.2 and doesn't require conda-forge.

**Comment by kjsanger** [comment]
Resolved by #195
